docs/BitcoinCoreKeys.md

```diff
@@ -38,85 +38,64 @@ This is an example of `eclair-signer.conf` configuration file:
 }
 ```
 
-### 3. Configure Eclair to handle private keys for this wallet
+### 3. Use Eclair to generate descriptors and import them into a new bitcoin wallet
 
-Set `eclair.bitcoind.wallet` to the name of the wallet in your `eclair-signer.conf` file (`eclair` in the example above) and restart Eclair.
-Eclair will automatically create a new, empty, descriptor-enabled, watch-only wallet in Bitcoin Core and import its descriptors.
+Restart eclair, without changing `eclair.bitcoind.wallet` (so it still uses the same bitcoin standard wallet).
```

Comment:
No this is a bit tedious, but users need to be able to start eclair first, with a standard wallet, so they can use its API to generate descriptors and then create a new wallet managed by eclair.

Reply:
Oh that's because we haven't created yet the bitcoind wallet so eclair would fail to start, got it.
It's still doesn't make sense in that case to say "the same bitcoin standard wallet" if the user is brand new.
We should probably do something like:

"Restart eclair, without changing `eclair.bitcoind.wallet` (so it uses the default wallet or the previously used bitcoin wallet for existing nodes)."

Reply:
done in https://github.com/ACINQ/eclair/pull/2746/commits/986d3f718ed45d889ac311c8190ba51564a08e2f